tournament/doc/tournament-api-documentation.md

```diff
@@ -45,13 +45,15 @@ Create a new tournament
 - Tournament name must be between 3 and 20 characters and can only contain alnum and space
 - Players must be between 2 and 16 (optional, default = 16 players)
 - Registration deadline (optional)
+- Nickname for the tournament (optional)
 - A boolean that specifies if tournament is private
 
 > ```javascript
 > {
 >     "name": "World Championship",
 >     "max-players": 16,
 >     "registration-deadline": "2024-02-17T10:53",
+>     "nickname": "Player",
```

Comment:
Use `username` instead of `nickname` to stay consistant in naming:
https://github.com/tdameros/42-transcendence/blob/6376487c4ef8fca33fd11c93f1bb3f4cf88544df/user_management/doc/User_management.md?plain=1#L23

Reply:
it's not exactly a username. The nickname can be different for each tournament.